docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -427,11 +427,11 @@ To download the diagnostics bundle for local viewing:
 . Click the **Request diagnostics .zip** button.
 +
 [role="screenshot"]
-image::images/collect-agent-diagnostics2.png[Collect agent diagnostics under agent details]
+image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agent details]
 
 Any in-progress or previously collected bundles for the {agent} will be listed on this page.
 
-Note that the bundles are stored in {es} and will be removed after 30 days.
+Note that the bundles are stored in {es} and will be removed after 30 days. You can also delete any previously created bundle by clicking the `trash can` icon.
```

Comment:
same correction needed here

Reply:
Thanks! Fixed in [2911cb9](https://github.com/elastic/ingest-docs/pull/1081/commits/2911cb925942d0b9df02138ac5cd7455c65857d2).